Sources/Editor.php

```diff
@@ -373,6 +498,180 @@ public static function getMessageIcons(int $board_id): array
 		return array_values($icons);
 	}
 
+	/*************************
+	 * Internal static methods
+	 *************************/
+
+	/**
+	 * Initializes default BBC tags for the toolbar.
+	 */
+	protected static function initBbcTags(): void
+	{
+		// The below array makes it dead easy to add images to this control. Add it to the array and everything else is done for you!
+		/*
+			array(
+				'code' => 'b', // Required
+				'description' => Lang::$editortxt['bold'], // Required
+				'image' => 'bold', // Optional
+				'before' => '[b]', // Optional
+				'after' => '[/b]', // Optional
+			),
+		*/
+		self::$bbc_tags = [
+			[
+				[
+					'code' => 'bold',
+					'description' => Lang::getTxt('bold', var: 'editortxt'),
+				],
+				[
+					'code' => 'italic',
+					'description' => Lang::getTxt('italic', var: 'editortxt'),
+				],
+				[
+					'code' => 'underline',
+					'description' => Lang::getTxt('underline', var: 'editortxt'),
+				],
+				[
+					'code' => 'strike',
+					'description' => Lang::getTxt('strikethrough', var: 'editortxt'),
+				],
+				[
+					'code' => 'superscript',
+					'description' => Lang::getTxt('superscript', var: 'editortxt'),
+				],
+				[
+					'code' => 'subscript',
+					'description' => Lang::getTxt('subscript', var: 'editortxt'),
+				],
+				[
+					'image' => 'tt',
+					'code' => 'tt',
+					'description' => Lang::getTxt('tt', var: 'editortxt'),
+				],
+				[
+					'image' => 'hidden',
+					'code' => 'spoiler',
+					'description' => Lang::getTxt('spoiler', var: 'editortxt'),
+				],
+				[],
+				[
+					'code' => 'pre',
+					'description' => Lang::getTxt('preformatted_text', var: 'editortxt'),
+				],
+				[
+					'code' => 'left',
+					'description' => Lang::getTxt('align_left', var: 'editortxt'),
+				],
+				[
+					'code' => 'center',
+					'description' => Lang::getTxt('center', var: 'editortxt'),
+				],
+				[
+					'code' => 'right',
+					'description' => Lang::getTxt('align_right', var: 'editortxt'),
+				],
+				[
+					'code' => 'justify',
+					'description' => Lang::getTxt('justify', var: 'editortxt'),
+				],
+				[],
+				[
+					'code' => 'font',
+					'description' => Lang::getTxt('font_name', var: 'editortxt'),
+				],
+				[
+					'code' => 'size',
+					'description' => Lang::getTxt('font_size', var: 'editortxt'),
+				],
+				[
+					'code' => 'color',
+					'description' => Lang::getTxt('font_color', var: 'editortxt'),
+				],
+				[],
+				[
+					'code' => 'removeformat',
+					'description' => Lang::getTxt('remove_formatting', var: 'editortxt'),
+				],
+			],
+			[
+				[
```

Comment:
```suggestion
				],
				[
```

Removes the extra group, letting the flex box handle it.

Reply:
This actually IS what triggers the breaking divider . Also, it'll probably break mods that expect the array structure to be like this. Although I've never seen any that specifically look for this index; instead, they rely on `count()`.